### ISSUE TITLE
unit test failure: FAIL: TestFromSif

### ISSUE THREAD

**Issue by jscook2345** [apptainer/singularity#3958]
This was seen on opensuse leap 15.0 while testing another PR. Please let me know if you have an idea to try or need more information. Thanks!

Information about the system:
```
$ uname -a
Linux vagrant-openSUSE-Leap 4.12.14-lp150.12.67-default #1 SMP Tue Jul 9 08:18:26 UTC 2019 (b610084) x86_64 x86_64 x86_64 GNU/Linux
```

```
=== RUN   TestFromSif
--- FAIL: TestFromSif (4.15s)
    bundle_linux_test.go:84: failed to find loop device: failed to attach image /tmp/busybox719733936: failed to set loop flags on loop device: resource temporarily unavailable
FAIL
coverage: 28.6% of statements
FAIL    github.com/sylabs/singularity/pkg/ocibundle/sif 4.162s
```

**Comment by cclerget** [comment]
@jscook2345 Same as #3860 , this is a kernel bug regression in loop device code

**Comment by jscook2345** [comment]
@cclerget Do we have a workaround for this or do we need to wait on the kernel update?

**Comment by cclerget** [comment]
@jscook2345 The only workaround for now is to downgrade the kernel
